ts/Series/Line/LineSeries.ts

```diff
@@ -4664,13 +4664,15 @@ class LineSeries {
 
             // Set the the plotY value, reset it for redraws
             // #3201
-            point.plotY = (
-                (typeof yValue === 'number' && yValue !== Infinity) ?
-                    limitedRange(yAxis.translate(
-                        yValue, 0 as any, 1 as any, 0 as any, 1 as any
-                    ) as any) :
-                    void 0
-            );
+            point.plotY = void 0;
+            if (typeof yValue === 'number' && yValue !== Infinity) {
```

Comment:
Use the `isNumber` utility function that checks that the type is number, that it is not Infinity or -Infinity, and not NaN.